snuba/snuba_migrations/generic_metrics/0053_counters_aggregated_add_sampling_weight.py

```diff
@@ -0,0 +1,58 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+from snuba.utils.schemas import AggregateFunction, Float
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    storage_set_key = StorageSetKey.GENERIC_METRICS_COUNTERS
+
+    local_table_name = "generic_metric_counters_aggregated_local"
+    dist_table_name = "generic_metric_counters_aggregated_dist"
+
+    columns: Sequence[tuple[Column[MigrationModifiers], str | None]] = [
+        (
+            Column(
+                "value_weighted",
+                AggregateFunction("sum", [Float(64)]),
+            ),
+            "value",
+        ),
+    ]
+
+    def forwards_ops(self) -> Sequence[SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=self.storage_set_key,
+                table_name=table_name,
+                column=column,
+                after=after,
```

Comment:
Nit: You can skip the after field

Reply:
Do we not want the weighted_value to be next to the regular value?

Reply:
It does make sense to add value_weighted after the value field in this case. Ignore my comment.